app/models/data_object.rb

```diff
@@ -80,7 +80,10 @@ class DataObject < ActiveRecord::Base
 
   scope :images, -> { where(data_type_id: DataType.image.id) }
   scope :texts,  -> { where(data_type_id: DataType.text.id) }
-
+  scope :articles_of_today, ->(current_user) {
+    DataObject.joins(:users_data_object).where(users_data_objects: {user_id: current_user.id},data_type_id: DataType.text.id,
```

Comment:
Line is too long. [125/80]<br>Use 2 (not 3) spaces for indentation.<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.